cms/models.py

```diff
@@ -461,6 +461,12 @@ def auto_save_related_posts_to_git(sender, instance, action, **kwargs):
 
 @receiver(post_save, sender=Post)
 def auto_save_post_to_git(sender, instance, created, **kwargs):
+    # When saving to git, we update the uuid of the instance to ensure
+    # we can find it again.
+    # We need to always retrieve the latest version of the instance to
+    # make sure we always have the uuid if it has been set.
+    instance = Post.objects.get(pk=instance.pk)
```

Comment:
Can we rather do this `if not instance.uuid`? This should only ever happen on first save, if I'm not mistaken.